### DISCUSSION TITLE
firebase error

### DISCUSSION THREAD

**Discussion by Kinjal260299** [community/community#59331] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Firebase error AbortError: Registration failed - could not retrieve the public key

this error occured only in chrome in chromium its working properly

**Comment by isaacriehm** [comment]
It seems as though Firebase may be attempting to use old cached login information. Try clearing your cookies.

**Comment by JoAzar** [comment]
Need a public key, if you use Linux need create a public key and use this in github for ssh transfer
